pact/verifier.py

```diff
@@ -28,68 +28,160 @@ def __str__(self):
-                                                    include_wip_pacts_since=include_wip_pacts_since,
-                                                    **options)
-
-        return success, logs
 
-    def verify_with_broker(self, enable_pending=False, include_wip_pacts_since=None, **kwargs):
+        return_code, logs = VerifyWrapper().call_verify(
+            *pacts,
+            provider=self.provider,
+            provider_base_url=self.provider_base_url,
+            enable_pending=enable_pending,
+            include_wip_pacts_since=include_wip_pacts_since,
+            **options
+        )
+
+        return return_code, logs
+
+    def verify_with_broker(
+            self,
+            *pacts,
+            enable_pending=False,
+            include_wip_pacts_since=None,
+            **kwargs
+    ):
         """Use Broker to verify.
 
-        Args:
-            broker_username ([String]): broker username
-            broker_password ([String]): broker password
-            broker_url ([String]): url of broker
-            enable_pending ([Boolean])
-            include_wip_pacts_since ([String])
-            publish_version ([String])
-
+        Usage:
+
+        >>> from pact import Verifier
+        >>>
+        >>> verifier = Verifier(
+        ...   provider='UserService',
+        ...   provider_base_url='http://localhost:5001'
+        ... )
+        >>>
+        >>> result, _ = verifier.verify_with_broker(
+        ...   'https://broker/consumer-provider.json',  # Optional
+        ...   enable_pending=True,
+        ...   broker_username='pactbroker',
+        ...   broker_password='pactbroker',
+        ...   broker_url='http://localhost',
+        ... )
+        >>>
+        >>> assert result == 0
+
+        :param pacts: Optional list of pacts to verify. Every pact in that list
+            can be an HTTP URI or a local file.
+        :type pacts: str
+        :param enable_pending: Allow pacts which are in pending state to be
+            verified without causing the overall task to fail.
+        :type enable_pending: bool
+        :param include_wip_pacts_since: Allow pacts that don't match given
+            consumer selectors (or tags) to  be verified, without causing the
+            overall task to fail
+        :type include_wip_pacts_since: str or None
+        :keyword str broker_username: Pact Broker basic auth username.
+        :keyword str broker_password: Pact Broker basic auth password.
+        :keyword str broker_token: Pact Broker bearer token.
+        :keyword str broker_url: Base URL of the Pact Broker from which to
+            retrieve the pacts.
+        :raises Exception: When `publish_verification_results` is set,
+            but `publish_version` is not provided.
+        :return: Returns a tuple of two elements. The first indicates the
+            status of the operation, so that 0 means success, and the second
+            contains the logs of the operation.
```

Comment:
Maybe a bit shorter:
```suggestion
        :return: Returns a tuple of the overall result (where 0 indicates success), and logs produced.
```